src/containers/catalog/CatalogActionFactories.js

```diff
@@ -19,4 +19,11 @@ export function createEntitySetListFailure(errorMessage: string) {
     type: actionTypes.ENTITY_SET_LIST_FAILURE,
     errorMessage
   }
+}
+
+export function createUpdateFilters(filterParams: {keyword: string, propertyTypeIds: string[], entityTypeId: string}) {
```

Comment:
`Object` works for now. I'd like to figure out a better way of defining props and params in general. Flow isn't triggered as part of our Babel builds. I'm hesitant to invest too much into it until we get that working.